### PR TITLE
update CLI Takedate

### PR DESCRIPTION
**Pull request by evoludolab**
* option --timestamp added to set create_at to timestamp of media file for media that lacks EXIF information
* use same formatting for 'sysdate' as for 'takedate'

For background see #870

### CODE REVIEW

**Review by kamil4 — APPROVED** [review]
This looks good. We should also provide this functionality during regular import, but that can be done in a separate PR.

File: app/Console/Commands/Takedate.php [hunk 1]
```diff
@@ -45,22 +50,36 @@ public function handle(Extractor $metadataExtractor)
...
-			if (file_exists($url)) {
-				$info = $metadataExtractor->extract($url, $photo->type);
-				if ($info['takestamp'] != null) {
-					$photo->takestamp = $info['takestamp'];
-					if ($photo->save()) {
-						$this->line($i . ': Takestamp updated for ' . $photo->title);
-					} else {
-						$this->line($i . ': Failed to update takestamp for ' . $photo->title);
-					}
+			$i++;
```
Comment (ildyria) [inline app/Console/Commands/Takedate.php:53]: This `$i++` is not consistent with what is actually being done and is off by +1.
Just place it before each `continue` and at the end of the loop.

File: app/Console/Commands/Takedate.php [hunk 2]
```diff
@@ -45,22 +50,36 @@ public function handle(Extractor $metadataExtractor)
...
-					$this->line($i . ': Could not get Takestamp data for ' . $photo->title . '.');
+					$this->line($i . ': Failed to update takestamp for ' . $photo->title);
 				}
+				continue;
+			}
+			if (!$timestamps) {
+				$this->line($i . ': Could not get Takestamp data for ' . $photo->title . '.');
+				continue;
+			}
+			$filename = $url;
```
Comment (ildyria) [inline app/Console/Commands/Takedate.php]: why is this needed? `$url` is not used anymore afterwards.

if you do not like `$url` then just rename all the previous `$url` before by `$filename`.

**Review by ildyria — CHANGES_REQUESTED** [review]
Looks good.
Untested.

Please fix the two comments. :smiley:

**Review by ildyria — APPROVED** [review]

**Review by cleytontiagosorinho — APPROVED** [review]
Só os que compensa no momento

**Review by cleytontiagosorinho — APPROVED** [review]
As melhores opicoes

**Pull request merged**